test/e2e/rollback_test.go

```diff
@@ -0,0 +1,54 @@
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	nmstatev1alpha1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1alpha1"
+)
+
+var _ = Describe("NodeNetworkState controller", func() {
+	var (
+		br1Absent = nmstatev1alpha1.State(`interfaces:
+  - name: br1
+    type: linux-bridge
+    state: absent
+`)
+
+		br1 = nmstatev1alpha1.State(`interfaces:
+  - name: br1
+    type: linux-bridge
+    state: up
+    bridge:
+      options:
+        stp:
+          enabled: false
+      port: []
+`)
+	)
+	Context("when there is an error at vlan-filtering", func() {
+		BeforeEach(func() {
+			By("Rename vlan-filtering to force failure after nmstatectl set")
```

Comment:
We do not fail on `nmstatectl set` though. "to force failure during state configuration" may be more precise?